### PR TITLE
Fix issue 117 weak test

### PR DESCRIPTION
**Pull request by chick**
Fixed weak test described Issue #117
Improved tests in accordance with suggestion by @WindProphet
Bumped version dependencies in build.sbt

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -39,8 +39,8 @@ scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-language:refle
 // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
 // The following are the default development versions, not the "release" versions.
 val defaultVersions = Map(
-  "chisel3" -> "3.1-SNAPSHOT",
-  "chisel-iotesters" -> "1.2-SNAPSHOT"
+  "chisel3" -> "3.2-SNAPSHOT",
```
Comment (ucbjrl) [inline build.sbt]: I was just about to do this, so thanks, but it should really be a separate pr.

File: src/test/scala/examples/RiscTests.scala [hunk 2]
```diff
@@ -47,7 +47,8 @@ class RiscTests(c: Risc) extends PeekPokeTester(c) {
 
 class RiscTester extends ChiselFlatSpec {
   behavior of "Risc"
-  backends foreach {backend =>
+//  backends foreach {backend =>
+  Seq("firrtl") foreach {backend =>
```
Comment (ucbjrl) [inline src/test/scala/examples/RiscTests.scala]: Are we dropping backends intentionally or is this a scalpel left in the patient?

Comment (chick) [reply src/test/scala/examples/RiscTests.scala]: Ouch!

File: build.sbt (hunk 1 above)
Comment (chick) [reply build.sbt]: OK
